winui/Kanban/getting-started.md

```diff
@@ -0,0 +1,273 @@
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+
+        taskDetail = new KanbanModel();
+        taskDetail.Title = "WF Issue";
+        taskDetail.Id = "1254";
+        taskDetail.Description = "HorizontalAlignment for tooltip is not working";
+        taskDetail.Category = "Review";
+        taskDetail.IndicatorColorKey = "High";
+        taskDetail.Tags = new List<string>() { "Bug fixing" };
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle5.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+        return taskDetails;
+    }
+
+    #endregion
+}
+
+{% endhighlight %}
+
+### Bind item source for Kanban
+
+To populate the Kanban card items, utilize the `ItemsSource` property of SfKanban.
+
+{% tabs %}
+
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="2" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines="2" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Automatically generating columns in the Kanban control
+
+To automatically generate columns in the Kanban control, you can set the `AutoGenerateColumns` property to `true`. When enabled, the columns are generated dynamically based on the `Category` property of KanbanModel, creating a column for each distinct value in the `ItemsSource`.
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 AutoGenerateColumns="True">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines="2" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+this.kanban.AutoGenerateColumns = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Manually defining columns in the Kanban control
+
+You can manually define columns in the Kanban control by adding `KanbanColumn` objects to the `Columns` collection property of SfKanban. The `ItemsSource` bound to the Kanban control will populate the respective columns based on the `Categories` property in `KanbanColumn`.
+
+The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3 4 5 6" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 AutoGenerateColumns="False">
+    <kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
```

Comment:
Update code

` <kanban:SfKanban x:Name="kanban"
                  AutoGenerateColumns="False"
                  ItemsSource="{Binding TaskDetails}">
     <kanban:KanbanColumn HeaderText="To Do" Categories="Open" />
     <kanban:KanbanColumn HeaderText="In Progress" Categories="In Progress" />
     <kanban:KanbanColumn HeaderText="Code Review" Categories="Code Review" />
     <kanban:KanbanColumn HeaderText="Done" Categories="Done" />
 </kanban:SfKanban>`